Sources/muterCore/TestReporting/HTML/HTMLReporter.swift

```diff
@@ -0,0 +1,264 @@
+                </header>
+                <main>
+                    <div class="summary">
+                        <p>In total, Muter introduced
+                            <span class="strong">\(testReport.totalAppliedMutationOperators)</span> mutants in
+                            <span class="strong">\(testReport.fileReports.count)</span> files.
+                        </p>
+                    </div>
+                    <div class="divider">
+                        <span class="divider-content">Mutation Operators per File</span>
+                    </div>
+                    <div class="mutation-operators-per-file">
+                        <div class="toggle">
+                            <input id="show-more-mutation-operators-per-file" type="checkbox" onclick="showHide(this.checked, 'mutation-operators-per-file');" />
+                            <label for="show-more-mutation-operators-per-file">Show all</label>
+                        </div>
+                        \(testReport.fileReports.scoreHTMLTable())
+                    </div>
+                    <div class="divider">
+                        <span class="divider-content">Applied Mutation Operators</span>
+                    </div>
+                    <div class="applied-operators">
+                        <div class="toggle">
+                            <input id="show-more-applied-operators" type="checkbox" onclick="showHide(this.checked, 'applied-operators');" />
+                            <label for="show-more-applied-operators">Show all</label>
+                        </div>
+                        \(testReport.fileReports.operatorsHTMLTable())
+                    </div>
+                </main>
+                <footer>
+                    <div class="footer">\(now().string)</div>
+                </footer>
+            </div>
+        </body>
+    </html>
+    """
+}
+
+private extension Date {
+    var string: String {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "EEEE, MMMM d yyyy HH:mm:ss"
+
+        return formatter.string(from: self)
+    }
+}
+
+private extension Array where Element == MuterTestReport.FileReport {
+    func scoreHTMLTable() -> String {
+        let thead =
+            """
+            <thead>
+                <tr>
+                    <th>File</th>
+                    <th># of Introduced Mutants</th>
+                    <th>Mutation Score</th>
+                </tr>
+            </thead>
+            """
+
+        let tbody = sorted().compactMap { report in
+            """
+            <tr>
+              <td class="left-aligned">\(report.fileName)</td>
+              <td class="right-aligned">\(report.appliedOperators.count)</td>
+              <td class="right-aligned" style="color: \(report.scoreColor);">\(report.mutationScore)</td>
+            </tr>
+            """
+        }.joined(separator: "\n")
+
+        return """
+        <table id=\"mutation-operators-per-file\">
```

Comment:
Do you need to escape quotes in multiline strings? If so, consider raw strings maybe?